pkgs/servers/tailscale/default.nix

```diff
@@ -1,16 +1,17 @@
-{ lib, stdenv, buildGoModule, fetchFromGitHub, makeWrapper, iptables, iproute2, procps }:
+{ lib, stdenv, buildGoModule, fetchFromGitHub, makeWrapper, iptables, iproute2
+, procps }:
```

Comment:
Since there's no change here can this be put back on one line?

Reply:
Sorry about that. Editor auto-formatted, and I missed it in the diff output :(